CONTRIBUTING.rst

```diff
@@ -124,6 +124,11 @@ These are the basic steps needed to start developing on Sphinx.
 
        PYTHONWARNINGS=all make test
 
+   * Arguments to pytest can be passed as TEST environment variable, e.g. in
```

Comment:
For reference, I'd like to rewrite this entire document to use tox instead, meaning this would become:

    tox -e py27 tests/test_module.py::test_new_feature

This doesn't mean this shouldn't go in as-is now though